pkg/parse/parse.go

```diff
@@ -129,10 +129,30 @@ func (p *Parser) Parse(filename string, fs afero.Fs) (*sysl.Module, error) {
 			return nil, err
 		}
 
-		antlr.NewParseTreeWalker().Walk(listener, tree)
+		localListener := NewTreeShapeListener()
```

Comment:
Why do you add another locallistener instead of checking listener.imports directly after `antlr.NewParseTreeWalker().Walk(listener, tree)`?

Reply:
The 'listener' is a cumulative queue of import dependencies originating from the original module (similar to how a BFS works) so it would also contain dependencies of it's parent module etc and not reflect the dependencies of just a single module (file). So I need to create a local listener to grab just the imports of a single file.